resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/submitsteps/PythonStep.scala

```diff
@@ -30,7 +30,7 @@ private[spark] class PythonStep(
     val resolvedOtherPyFilesString = if (otherPyFiles.isEmpty) {
       "null"
     } else {
-      otherPyFiles.mkString(",")
+      KubernetesFileUtils.resolveSubmittedUris(otherPyFiles, filesDownloadPath).mkString(",")
```

Comment:
Does `PythonRunner` handle files with URI schemes of `hdfs` or other remote locations? If not, then we probably want to use the downloaded path for every URI - hence we should use `KubernetesFileUtils.resolveFilePaths` instead.